### ISSUE TITLE
Masked Language Model in English not working

### ISSUE THREAD

**Issue by ontocord** [nghuyong/ERNIE-Pytorch#31]
I tried the chinese masekd langauge model ernie-1.0 and it worked, but ernie-2.0-en does not work. See code below.

```

from torch import nn
from transformers import AutoModelForMaskedLM, AutoTokenizer
from transformers import AutoTokenizer, AutoModel, BertForMaskedLM, BertTokenizer
tokenizer = BertTokenizer.from_pretrained("nghuyong/ernie-2.0-en")
model = BertForMaskedLM.from_pretrained("nghuyong/ernie-2.0-en")

input_txt = ["[MASK] was an American [MASK]  and lawyer who served as the 16th president  of the United States from 1861 to 1865. [MASK] led the nation through the American Civil War, the country's greatest [MASK], [MASK], and [MASK] crisis. ", \
             "[MASK], who served as the first  president of the United States from [MASK] to 1797, was an American political leader, [MASK] [MASK], statesman, and Founding Father. Previously, he led Patriot forces to [MASK] in the nation's War for Independence. ", \
             "[MASK], the first African-American [MASK] of the [MASK] [MASK], is an American politician and attorney who served as the 44th [MASK] of the United States from [MASK] to 2017.  [MASK] was a member of the [MASK] [MASK]. "]
#input_txt = 
input_txt= [i.replace("[MASK]", tokenizer.mask_token) for i in input_txt] #
inputs = tokenizer(input_txt, return_tensors='pt', add_special_tokens=True, padding=True)
inputs['output_attentions'] = True
inputs['output_hidden_states'] = True
inputs['return_dict'] = True
#model = model.eval()
#print (type(model))
#model.config.num_hidden_layers=6
outputs = model(**inputs)
print ((outputs.keys()))
if True:
  predictions = outputs.logits
  for pred in predictions:
    print ("**")
    sorted_preds, sorted_idx = pred.sort(dim=-1, descending=True)
    for k in range(2):
        predicted_index = [sorted_idx[i, k].item() for i in range(0,len(predictions[0]))]
        predicted_token = ' '.join([tokenizer.convert_ids_to_tokens([predicted_index[x]])[0] for x in range(1,len(predictions[0]))]).replace('Ġ', ' ').replace('  ', ' ').replace('##', '')
        print(predicted_token)
```

Results in the following output:

odict_keys(['logits', 'hidden_states', 'attentions'])
**
fold pew pew tery peer itated orth unk nya kindergarten mate tomb tery pew mate ipa peer pew nya nya tery pew fold pew mate cie its mate tery unk pew pew mate istan nowhere nowhere nowhere house mate nowhere mate mate nowhere nowhere mate mg fold fold fold fold
lead unopposed nya nobel olate aus fold pew ung ennial pew entation unk tub ile tery tery suit tee cape alison mate nowhere 501 ile cola gaza zell auschwitz wed 3a vent chai ausen ught ved som fold agi itated tery som dre recess pew ibe lead nate nowhere nowhere
**
lead pew zee nya ennial lain tery tery tub gael ipa fold pew ion ʂ fen pew pew pew tery bethel gina stituting rayon lau icon tto pew nowhere tana 283 pew paste pew pew ung fold unk mate bor rink mate pass pew mate pass lain pass mate mg
hesis ith chia mobile kindergarten zell fold unk pew propose fold peer ia [ nya khan ith ith cree pass nya nya aea mental pew borrow nay ivo itated pass elt mate tables cie fold pew pass ny submit paste ny zell cell nowhere ference iss pew hey pew rai
**
cree pew phy elt lain ail ail oft yst cycle rink popcorn pew shah shah tery nya istan istan oblast nya kindergarten cycle zko pment householder cycle aton pment anytime apparent nya taste ctic fold panchayat nya sy thee thee nap pointe mate mg nx fold fold fold fold fold
riam mori rys nowhere unk granny inn zee oche theirs visa bi cree pew nya nobel shah unk agne islamabad shah panchayat game entation quent cycle erian pment cola istan nya zel trivial inium cree nowhere thee thee thou wide ke hesis inca mation ett ett ety root ett ett

**Comment by nghuyong** [comment]
How about the result of your test cases on the original paddlepaddle version ?

**Comment by ontocord** [comment]
I haven't tried it with paddlepaddle. Do you think the original Ernie implementation doesn't work?

**Comment by BigSalmon2** [comment]
@ontocord Did you figure it out?

**Comment by MrZhengXin** [comment]
It seems that it's still not working?
Wondering if MLMHead is correctly loaded...

**Comment by nghuyong** [comment]
I checked the [Baidu's released model params](https://github.com/PaddlePaddle/ERNIE) and don't found **any mlm related params**, so the last mlm layer is **inited by random params**
